assets/js/script.js

```diff
@@ -1,37 +1,37 @@
 $(document).ready(function() {
   // hide contact us form
-  if( !window.location.href.match(/localhost/) ) {
+  if (!window.location.href.match(/localhost/)) {
     var interval = setInterval(function() {
       var contactUs = $('body').find('img[src*="icf.improvely.com"]')
-      if( !contactUs.length ) { return; }
+      if (!contactUs.length) { return; }
       contactUs.hide()
       clearInterval(interval)
     }, 200)
   }
 
   // fix odd safari bug where the delegated click is not being recognized, but only for the bag
   setTimeout(function() {
-    $('.js-bag').on('click', function(){})
+    $('.js-bag').on('click', function() {})
   }, 1000)
 
   // get template html and embed it in the page
   fetch((window.baseUrl || '') + '/assets/html/custombar.html').then(function(response) {
     return response.text()
   }).then(function(html) {
     html = html.replace(/assets\/img\//g, (window.baseUrl || '') + 'assets/img/')
-    $('#custombar').parent().css({position: 'relative', overflow: 'hidden'})
+    $('#custombar').parent().css({ position: 'relative', overflow: 'hidden' })
     $('#custombar').before(html).remove()
     render()
   })
 })
 
 var state = {
-  bag:          'black',
-  letters:      'AA',
-  materials:    ['white', 'white'],
+  bag: 'black',
```

Comment:
maybe ill make it an object that has a color and style because were gunna need multiple styles anyways...could also store the dimensions in there too